package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "style-loader",
-  "version": "0.13.1",
+  "version": "0.14.0",
```

Comment:
Please don't 😛 .  `npm version` is the reponsibility of the respective maintainer